ansible/roles/repo/tasks/redhat.yaml

```diff
@@ -12,6 +12,20 @@
     - lookup('env', 'RHSM_PASS') | length > 0
     - ansible_distribution == 'RedHat'
 
+- name: create proxy file
+  file:
+    path: /etc/yum.conf
+    state: touch
+  when: http_proxy is defined or https_proxy is defined
+
+- name: modify proxy settings if http_proxy is present
+  lineinfile:
+    path: /etc/yum.conf
+    line: "proxy={{ http_proxy }} #set by KIB"
```

Comment:
Don't we need to also set `https_proxy`?

Reply:
It looks like it just has proxy ?

https://linux.die.net/man/5/yum.conf